packages/core/src/provider/base.ts

```diff
@@ -87,7 +96,6 @@ export type SmartAccountProviderConfig<
 > = {
   rpcProvider: string | PublicErc4337Client<TTransport>;
   chain: Chain;
-  entryPointAddress: Address;
```

Comment:
see above re: question on why we can't just make this optional with a default value,